### PR TITLE
give more context about the project

### PR DESCRIPTION
**Pull request by bensigelman**

### CODE REVIEW

**Comment by bensigelman** [comment]
(per #9)

btw, I briefly looked into the blossom contributors thing... I ran into some headaches with custom js in jekyll and gave up for now.

**Comment by yurishkuro** [comment]
lgtm

**Comment by adriancole** [comment]
:shipit:

File: _config.yml [hunk 1]
```diff
@@ -26,8 +28,8 @@ permalink:        pretty
 
 # Setup
 title:            The OpenTracing Project
-tagline:          'Common APIs for distributed tracing and context propagation'
-description:      'Common APIs for distributed tracing and context propagation'
+tagline:          'Safe, general APIs for distributed tracing and context propagation'
+description:      'Safe, general APIs for distributed tracing and context propagation'
```
Comment (yurishkuro) [inline _config.yml]: "safe" in what sense?

Would be nice the first word to emphasize the interop property of the API.

Comment (bensigelman) [inline _config.yml]: yeah, "safe" is the wrong word... I changed this and also added a little more detail to the "APIs" page.

Comment (bensigelman) [inline _config.yml]: (btw, I meant "safe" in that they wouldn't get locked in)

File: _config.yml [hunk 2]
```diff
@@ -26,8 +28,8 @@ permalink:        pretty
 
 # Setup
 title:            The OpenTracing Project
-tagline:          'Common APIs for distributed tracing and context propagation'
-description:      'Common APIs for distributed tracing and context propagation'
+tagline:          'Consistent and expressive APIs for distributed tracing and context propagation'
+description:      'Consistent and expressive APIs for distributed tracing and context propagation'
```
Comment (yurishkuro) [inline _config.yml]: > (btw, I meant "safe" in that they wouldn't get locked in)

Vendor-neutral might be a good descriptor.

Comment (bensigelman) [inline _config.yml]: Sure, I'll add that to my next PR on the docs stuff.

Comment (adriancole) [inline _config.yml]: +1 vendor neutral

Comment (bensigelman) [inline _config.yml]: yeah, that fix is pending in my next round of updates... will send that out later today for sure. (yesterday was lost to non-laptop work)

**Pull request merged**